scripts/verify-replace.sh

```diff
@@ -16,26 +22,58 @@ do
     echo                                                                     
     echo =========================================================================================                                            
     repo_path=${BASE_REPO_PATH}/$(basename ${repo})
+    err_file=$(mktemp ${BASE_REPO_PATH}/$(basename ${repo})-error.XXX)
+    echo "error output file : ${err_file}"
+    std_out_file=$(mktemp ${BASE_REPO_PATH}/$(basename ${repo})-output.XXX)
+    echo "std output file : ${std_out_file}"
     echo "Cloning repo in /tmp"
     git clone --depth=1 ${repo} ${repo_path}
     echo "Repo cloned successfully"
     cd ${repo_path}
-    if ! make pre-verify; then
-        ERRORLIST+="($(basename ${repo}))"
+    make pre-verify 2> >(tee ${err_file})
+    rc=$?
+    if [ ${rc} -ne 0 ]; then
+        ERROR_REPO_LIST+="$(basename ${repo}) "
+        ERROR_FILE_LIST+="${err_file}  "
         continue
     fi
     echo "Initiating 'go mod replace' of current toolchain common version in dependent repos"
     go mod edit -replace github.com/codeready-toolchain/toolchain-common=${C_PATH}
-    make verify-dependencies || ERRORLIST+="($(basename ${repo}))"
+    make verify-dependencies 2> >(tee ${err_file}) 1> >(tee ${std_out_file})
+    rc=$?
+    if [ ${rc} -ne 0 ]; then
+    ERROR_REPO_LIST+="$(basename ${repo}) " 
+    ERROR_FILE_LIST+="${err_file}  "
+    STD_OUT_FILE_LIST+="${std_out_file} "
+    fi
     echo                                                          
     echo =========================================================================================
     echo                                                           
 done
-if [ ${#ERRORLIST[@]} -ne 0 ]; then
+echo                "Summary"
+if [ ${#ERROR_REPO_LIST[@]} -ne 0 ]; then
     echo "Below are the repos with error: "
-    for e in ${ERRORLIST[*]}
+    for error_repo_name in ${ERROR_REPO_LIST[*]}
     do
-        echo "${e}"
+        for error_file_name in ${ERROR_FILE_LIST[*]}
+        do
+            if [[ ${error_file_name} =~ ${error_repo_name} ]]; then 
+                echo                                                          
+                echo =========================================================================================
+                echo 
+                echo                       "${error_repo_name} has the following errors "
+                echo                                                          
+                echo =========================================================================================
+                echo 
+                cat "${error_file_name}"
+            fi
+        done
+        for std_out_file_name in ${STD_OUT_FILE_LIST[*]}
+            do
+                if [[ ${std_out_file_name} =~ ${error_repo_name} ]]; then 
+                    cat "${std_out_file_name}" | grep  "${GO_LINT_REGEX}\|${ERROR_REGEX}"
```

Comment:
from the last failure (in member-operator), it's apparent that the selection doesn't include everything - eg it's missing the line of the failed test (which is really missing to understand the problem)
```
                        	            				/tmp/replace-verify.OGM/member-operator/controllers/idler/idler_controller_test.go:259
```
https://github.com/codeready-toolchain/toolchain-common/actions/runs/13068368416/job/36464570446?pr=450#step:5:662

I would add `-C 5` to make sure that the output contains the whole context
```suggestion
                    cat "${std_out_file_name}" | grep -C 5  "${GO_LINT_REGEX}\|${ERROR_REGEX}"
```